### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Wednesday, 8th of January 2025, 2:30pm UTC

### DISCUSSION THREAD

**Discussion by tormath1** [flatcar/Flatcar#1598] [Flatcar Office Hours]
In the office hours we discuss news and happenings in and around Flatcar, provide a space for our user community to discuss with maintainers, show demos, and do our release planning.

- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording: https://www.youtube.com/live/Gxayfnzk3uc
  - All recordings: https://www.youtube.com/@FlatcarLinux/live
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
  
## Welcome
- Brief intro/check-in of the new participants joining the call. Please introduce yourself and share what brings you here today.

### Flatcar at [FOSDEM 2025](https://fosdem.org/) - February 1+2, Brussels
- We have a shared booth with Gentoo Linux!
- Presentations / talks:
  - [Flatcar and Gentoo sitting in a tree - A collaboration of distributions](https://fosdem.org/2025/schedule/event/fosdem-2025-4763-flatcar-and-gentoo-sitting-in-a-tree-a-collaboration-of-distributions/)
  - [Add RISC-V support to your favorite Operating System](https://fosdem.org/2025/schedule/event/fosdem-2025-6036-add-risc-v-support-to-your-favorite-operating-system/)
  - [Immutable All the Way Down - using System Extensions to ship Kubernetes](https://fosdem.org/2025/schedule/event/fosdem-2025-4677-immutable-all-the-way-down-using-system-extensions-to-ship-kubernetes/)
- Our booth will participate in Chris Adams' CO<sub>2</sub> monitoring initiative; more information here: https://mastodon.social/@mrchrisadams/113781301236589758

### End of support for CGroupsV1 in early 2025
<details><summary>Flatcar will stop supporting the cgroups v1 backwards compatibility mode in Q1 2025.</summary>

- With our adoption of systemd-256, support for legacy CGroupsV1 will end in **Alpha, Beta, and eventually Stable**.
- [Enabling legacy CGroupsV1 during deployment](https://www.flatcar.org/docs/latest/container-runtimes/switching-to-unified-cgroups/#starting-new-nodes-with-legacy-cgroups) will not be supported anymore.
- Nodes that use CGroupsV1 legacy mode will *fail to update*. This ensures your workloads will not be disrupted. Enable CGroupsV2 on your legacy CGroupsV1 nodes in order to successfully update.
- LTS-2024 will support CGroupsV1 until late 2025.
</details>

## Agenda items
- Flatcar running on RISC-V (QEMU)

### Release Planning

- https://github.com/orgs/flatcar/projects/7/views/8

### Administrative tasks after meeting concluded
- [x] Create a new Github discussion for the next Flatcar Office Hours: https://github.com/flatcar/Flatcar/discussions/new?category=flatcar-office-hours
- [x] Close this discussion as "Resolved".

**Comment by ader1990** [comment]
Hello, I would like to present the Proof of Concept on RISC-V architecture, a fully working RISC-V Flatcar image booted using QEMU.
